src/main/drivers/system.c

```diff
@@ -151,11 +151,19 @@ void systemInit(void)
     cachedRccCsrValue = RCC->CSR;
     RCC_ClearFlag();
 
-
     enableGPIOPowerUsageAndNoiseReductions();
 
-
 #ifdef STM32F10X
+    // Set USART1 TX (PA9) to output and high state to prevent a rs232 break condition on reset.
+    // See issue https://github.com/cleanflight/cleanflight/issues/1433
+    gpio_config_t gpio;
+
+    gpio.mode = Mode_Out_PP;
+    gpio.speed = Speed_2MHz;
+    gpio.pin = Pin_9;
+    gpioInit(GPIOA, &gpio);
```

Comment:
Technically you should set PIN state first, then change it to output. There may be very short low-level pulse on output otherwise that may cause problems (it can theoretically generate false  startbit, and thus spurious 0xff byte, but most UART implementations have digital filter).